src/java.base/share/classes/jdk/internal/module/ModuleBootstrap.java

```diff
@@ -869,6 +873,21 @@ private static void addIllegalAccess(String illegalAccess,
         builder.complete();
     }
 
+    private static void addEnableNativeAccess(ModuleLayer layer) {
```

Comment:
It would be useful to add a method description in the same style as the existing methods, just to keep these methods consistent.

Reply:
and this

Reply:
I'll fix